plugins/bcli.c

```diff
@@ -683,6 +683,14 @@ static void json_add_feerate(struct json_stream *result, const char *fieldname,
 			     const struct estimatefees_stash *stash,
 			     uint64_t value)
 {
+	/* Anthony Towns reported signet had a 900kbtc fee block, and then
+	 * CLN got upset scanning feerate.  It expects a u32. */
+	if (value > 0xFFFFFFFF) {
+		plugin_log(cmd->plugin, LOG_UNUSUAL,
+			   "Feerate %"PRIu64" is ridiculous: trimming to 32 bites",
+			   value);
+		value = 0xFFFFFFFF;
+	}
```

Comment:
Nice! iirc there is a fixme somewhere that point that bcli is storing a u32 but lightningd is storing a u64 that this should be fine in this case? 

maybe just recalling the FIXME that we can solve this hack by upgrade the bcli to u64?